flowgate-api/src/main/java/com/vmware/flowgate/service/AssetService.java

```diff
@@ -151,6 +155,31 @@
          case MetricName.PDU_VOLTAGE:
             data.setMetricName(String.format(MetricKeyName.SERVER_CONNECTED_PDUX_OUTLETX_VOLTAGE, pduAssetId, value.getExtraidentifier()));
             result.add(data);
```

Comment:
if(server_voltage is empty){
server_voltage = value;
}
if(outlet){
outlet.v is not empyt . 
server_voltage = outelet.v
}